deployment/CP04OSSM_Deploy.csv

```diff
@@ -1,80 +1,80 @@
+KN-224,,AT-27,,CP04OSSM-RIC21-00-CPMENG000,1,1,2014-12-12T21:47:00,2015-05-10T19:57:00,CGMCP-04OSSM-00001,,CGCON-NCPM02-00004,39.93617,-70.8803,,7,455,
+KN-224,,AT-27,,CP04OSSM-RID26-00-DCLENG000,1,1,2014-12-12T21:47:00,2015-05-10T19:57:00,CGMCP-04OSSM-00001,,CGCON-NDCL26-00013,39.93617,-70.8803,,7,455,
+KN-224,,AT-27,,CP04OSSM-RID26-04-VELPTA000,1,1,2014-12-12T21:47:00,2015-05-10T19:57:00,CGMCP-04OSSM-00001,,CGINS-VELPTA-11417,39.93617,-70.8803,,7,455,
+KN-224,,AT-27,,CP04OSSM-RID26-06-PHSEND000,1,1,2014-12-12T21:47:00,2015-05-10T19:57:00,CGMCP-04OSSM-00001,,CGINS-PHSEND-P0098,39.93617,-70.8803,,7,455,
+KN-224,,AT-27,,CP04OSSM-RID26-07-NUTNRB000,1,1,2014-12-12T21:47:00,2015-05-10T19:57:00,CGMCP-04OSSM-00001,,CGINS-NUTNRB-00266,39.93617,-70.8803,,7,455,
+KN-224,,AT-27,,CP04OSSM-RID26-08-SPKIRB000,1,1,2014-12-12T21:47:00,2015-05-10T19:57:00,CGMCP-04OSSM-00001,,CGINS-SPKIRB-00271,39.93617,-70.8803,,7,455,
+KN-224,,AT-27,,CP04OSSM-RID27-00-DCLENG000,1,1,2014-12-12T21:47:00,2015-05-10T19:57:00,CGMCP-04OSSM-00001,,CGCON-NDCL27-00004,39.93617,-70.8803,,7,455,
+KN-224,,AT-27,,CP04OSSM-RID27-01-OPTAAD000,1,1,2014-12-12T21:47:00,2015-05-10T19:57:00,CGMCP-04OSSM-00001,,CGINS-OPTAAD-00165,39.93617,-70.8803,,7,455,
+KN-224,,AT-27,,CP04OSSM-RID27-02-FLORTD000,1,1,2014-12-12T21:47:00,2015-05-10T19:57:00,CGMCP-04OSSM-00001,,CGINS-FLORTD-01221,39.93617,-70.8803,,7,455,
+KN-224,,AT-27,,CP04OSSM-RID27-03-CTDBPC000,1,1,2014-12-12T21:47:00,2015-05-10T19:57:00,CGMCP-04OSSM-00001,,CGINS-CTDBPE-50004,39.93617,-70.8803,,7,455,
+KN-224,,AT-27,,CP04OSSM-RID27-04-DOSTAD000,1,1,2014-12-12T21:47:00,2015-05-10T19:57:00,CGMCP-04OSSM-00001,,CGINS-DOSTAD-00375,39.93617,-70.8803,,7,455,
+KN-224,,AT-27,,CP04OSSM-SBC11-00-CPMENG000,1,1,2014-12-12T21:47:00,2015-05-10T19:57:00,CGMCP-04OSSM-00001,,CGCON-BCPM01-00006,39.93617,-70.8803,,0,455,
+KN-224,,AT-27,,CP04OSSM-SBD11-00-DCLENG000,1,1,2014-12-12T21:47:00,2015-05-10T19:57:00,CGMCP-04OSSM-00001,,CGCON-BDCL11-00006,39.93617,-70.8803,,0,455,
+KN-224,,AT-27,,CP04OSSM-SBD11-01-MOPAK0000,1,1,2014-12-12T21:47:00,2015-05-10T19:57:00,CGMCP-04OSSM-00001,,CGINS-MOPAK0-99999,39.93617,-70.8803,,0,455,
+KN-224,,AT-27,,CP04OSSM-SBD11-02-HYDGN0000,1,1,2014-12-12T21:47:00,2015-05-10T19:57:00,CGMCP-04OSSM-00001,,CGINS-HYDGN0-99999,39.93617,-70.8803,,0,455,
+KN-224,,AT-27,,CP04OSSM-SBD11-06-METBKA000,1,1,2014-12-12T21:47:00,2015-05-10T19:57:00,CGMCP-04OSSM-00001,,CGINS-METLGR-00016,39.93617,-70.8803,,0,455,
+KN-224,,AT-27,,CP04OSSM-SBD12-00-DCLENG000,1,1,2014-12-12T21:47:00,2015-05-10T19:57:00,CGMCP-04OSSM-00001,,CGCON-BDCL12-00006,39.93617,-70.8803,,0,455,
+KN-224,,AT-27,,CP04OSSM-SBD12-03-HYDGN0000,1,1,2014-12-12T21:47:00,2015-05-10T19:57:00,CGMCP-04OSSM-00001,,CGINS-HYDGN0-99999,39.93617,-70.8803,,0,455,
+KN-224,,AT-27,,CP04OSSM-SBD12-04-PCO2AA000,1,1,2014-12-12T21:47:00,2015-05-10T19:57:00,CGMCP-04OSSM-00001,,CGINS-PCO2AA-33184,39.93617,-70.8803,,0,455,
```

Comment:
Reviewed correction to recovery time, sensor UIDs and deployment depths. Looks good.